mtgjson5/resources/additional_sets.json

```diff
@@ -88,5 +88,18 @@
         "nonfoil_only": true,
         "foil_only": false,
         "search_uri": "https://api.scryfall.com/cards/search?include_extras=true&include_variations=true&order=set&q=e%3Amb1&unique=prints"
+    },
+    "FWB": {
+        "name": "Foreign White Border",
+        "code": "FWB",
+        "released_at": "1994-04-11",
+        "set_type": "core",
+        "parent_set_code": "3ed",
+        "digital": false,
+        "nonfoil_only": true,
+        "foil_only": false,
+        "block_code": "lea",
```

Comment:
if parent code is `3ed`, is it correc that the block code is `lea`? should it not be `fbb` since it contains all the fbb cards just in white border?